06-platform-instruction.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+menutitle: PLATFORM 
+permalink: /platform/
+---
+
+# Provide platform information
+
+Submissions to ReScience come with code. We ask information about the platform that was used
+to run the code, the detail depending on the tools and programming languages used.
+
+This information will be included in the file `code/README.md` of the submission. Consider
+the whole execution pipeline for your results for possible additional tools (e.g. Bash,
+random number generator, etc).
+
+## Version information for Python
+
+For Python, NumPy, SciPy and matplotlib (common Python dependencies), the
+following script will provide detailed information on your platform.
+```py
+from __future__ import print_function
+import sys
+import numpy as np
+import scipy
+import matplotlib
+import platform
+
+print("Platform:", sys.platform)
+print("Python:", sys.version)
+print("Machine and architecture", platform.machine(), *platform.architecture())
+print("NumPy:", np.version.version)
+print("SciPy:", scipy.version.version)
+print("matplotlib:", matplotlib.__version__)
+```
+
+The output, for instance
+
+```
+Platform: linux
+Python: 3.5.2+ (default, Sep 22 2016, 12:18:14) 
+[GCC 6.2.0 20160914]
+Machine and architecture x86_64 64bit ELF
+NumPy: 1.12.0
+SciPy: 0.18.1
+matplotlib: 2.0.0
+```
+
+The version of any library that is not part of Python's standard library should also be
+given.
+
+## Version information for compiled languages
+
+For compiled languages, the important information consists in the platform, the
+compiler, its version, and the compiler flags.
+
```

Comment:
Add something like "The following scripts are examples for how the relevant information can be obtained, please adapt to your system."